com.chartboost.helium/Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/IRewardedEventListener.kt

```diff
@@ -0,0 +1,14 @@
+package com.chartboost.heliumsdk.unity
+
+/**
+ * Interface for sending real-time rewarded events
+ */
+interface IRewardedEventListener {
+    fun DidLoadRewarded(placementName: String, errorCode: Int, errorDescription: String)
+    fun DidShowRewarded(placementName: String, errorCode: Int, errorDescription: String)
+    fun DidCloseRewarded(placementName: String, errorCode: Int, errorDescription: String)
+    fun DidClickRewarded(placementName: String, errorCode: Int, errorDescription: String)
+    fun DidRecordImpression(placementName: String, errorCode: Int, errorDescription: String)
+    fun DidWinBidRewarded(placementName: String, auctionId: String, partnerId: String, price: Double)
+    fun DidReceiveReward(placementName: String, reward: Int)
```

Comment:
We want the function names to start with a lowercase letter: https://kotlinlang.org/docs/coding-conventions.html#function-names.

Reply:
Since this is a shared interface with C# I did not consider this, but I will make the change for consistency.